### ISSUE TITLE
Wildcard DNS record failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#547]
```
Issue running command [gcloud dns record].

What steps will reproduce the problem?
gcloud dns record-sets transaction execute --zone examplecom [Attached the 
transaction.yaml file]

What is the expected output? What do you see instead?

Expected output: <A success ooutput>
Actual Output:
```

ERROR: gcloud crashed (ScannerError): while scanning an alias
  in "transaction.yaml", line 17, column 9
expected alphabetic or numeric character, but found '.'
  in "transaction.yaml", line 17, column 10

```

Please provide any additional information below.
I was following this documentation: 
http://docs.deis.io/en/latest/installing_deis/gce/#configure-dns

Traceback (most recent call last):
  File "lib/googlecloudsdk/gcloud_main.py", line 169, in main
    gcloud_cli.Execute()
  File "lib/googlecloudsdk/calliope/cli.py", line 646, in Execute
    result = args.cmd_func(cli=self, args=args)
  File "lib/googlecloudsdk/calliope/backend.py", line 1402, in Run
    resources = command_instance.Run(args)
  File "lib/googlecloudsdk/api_lib/dns/util.py", line 58, in CatchHTTPErrorRaiseHTTPException
    return func(*args, **kwargs)
  File "lib/surface/dns/record_sets/transaction/execute.py", line 50, in Run
    change = transaction_util.ChangeFromYamlFile(trans_file)
  File "lib/googlecloudsdk/api_lib/dns/transaction_util.py", line 72, in ChangeFromYamlFile
    change_dict = yaml.safe_load(yaml_file)
  File "./lib/third_party/yaml/__init__.py", line 93, in safe_load
    return load(stream, SafeLoader)
  File "./lib/third_party/yaml/__init__.py", line 71, in load
    return loader.get_single_data()
  File "./lib/third_party/yaml/constructor.py", line 37, in get_single_data
    node = self.get_single_node()
  File "./lib/third_party/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "./lib/third_party/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "./lib/third_party/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
[output truncated]

```

Original issue reported on code.google.com by `dhilipsiva` on 16 Feb 2016 at 9:33

Attachments:
- [transaction.yaml](https://storage.googleapis.com/google-code-attachments/google-cloud-sdk/issue-547/comment-0/transaction.yaml)

**Comment by GoogleCodeExporter** [comment]
```
Apologies, this is a very silly mistake on my side. Its a YAML parser issue. 
Should have put wildcard domain in side double quotes. That solved my issue. 
Please close this Issue.
```

Original comment by `dhilipsiva` on 16 Feb 2016 at 9:39

**Comment by GoogleCodeExporter** [comment]
```
Thanks for following up!
```

Original comment by `z...@google.com` on 16 Feb 2016 at 6:26
- Changed state: **WorkingAsIntended**

**Comment by GoogleCodeExporter** [comment]
```
Actually, it's worth fixing up the error message on our end. We'll leave this 
open to track that.
```

Original comment by `z...@google.com` on 16 Feb 2016 at 6:27
- Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
Original comment by `z...@google.com` on 18 Feb 2016 at 6:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue should be fixed in the to-be-released Cloud SDK 99.0.0 version.
```

Original comment by `z...@google.com` on 22 Feb 2016 at 7:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
99.0.0 is out. Please re-open if you're still experiencing this issue.
```

Original comment by `z...@google.com` on 4 Mar 2016 at 6:09
- Changed state: **Fixed**
